packages/sources/kaiko-test/src/config/index.ts

```diff
@@ -0,0 +1,13 @@
+export const DEFAULT_INTERVAL = '1m'
+export const DEFAULT_SORT = 'desc'
+export const DEFAULT_MILLISECONDS = 1800000
+export const DEFAULT_ENDPOINT = 'trades'
+export const DEFAULT_API_ENDPOINT = 'https://us.market-api.kaiko.io/v2/data/trades.v1'
```

Comment:
Let's not use a separate var for this, it shouldn't be used outside of the adapter index so better to hardcode the value in the adapter obj

Reply:
Same thing goes for the others actually, they shouldn't be used other places